app/src/main/java/fr/free/nrw/commons/category/CategoryClient.java

```diff
@@ -37,7 +37,7 @@ public CategoryClient(CategoryInterface CategoryInterface) {
      * @return
      */
     public Observable<String> searchCategories(String filter, int itemLimit, int offset) {
-        return responseToCategoryName(CategoryInterface.searchCategories(filter, itemLimit, offset));
+        return responseToCategoryName(CategoryInterface.searchCategories("intitle:"+filter, itemLimit, offset));
```

Comment:
Can this be moved into CategoryInterface?

Reply:
@macgills  i tried to do that but i was facing problem with one thing, i read the doc for `@Query` but found nothing by which i could insert intitle in the same line of CategoryInterface, should i make a different function for adding initile(in the CategoryInterface file) which would call the @Query handling function

Reply:
Ah I see what is happening now, was confused.... yeah I don't think Retrofit has support for prefixing the values of query parameters because that probably shouldn't be a thing. Resolved